simple/src/codingPractice/SubstringBreaker.java

```diff
@@ -0,0 +1,44 @@
+package codingPractice;
+import java.util.HashSet;
```

Comment:
add space between package and import statement

Reply:
Done